Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt

```diff
@@ -125,7 +125,11 @@ fun CourseDetails(
                     orientation = Orientation.Vertical
                 )
             ) {
-                val openFraction = -sheetState.offset.value / dragRange
+                val openFraction = if (sheetState.offset.value.isNaN()) {
```

Comment:
Does `isNan()` ever happen?

Reply:
Yes, `Swipeable` changed to [initialize this offset with `NaN`](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-master-dev:compose/material/material/src/commonMain/kotlin/androidx/compose/material/Swipeable.kt;l=95) so this actually fixes a run time crash.